wikitemplate-IPFS.md

```diff
@@ -212,7 +212,7 @@
 
 ## IPNS Keys
 
-- [ ] Prerequisites: local node launched and local gateway configured. Go to `Settings -> IPFS`, there should be an available `Set up your IPNS keys` option, which opens `brave://settings/ipfs/keys`
+- [ ] Prerequisites: Brave local node launched and local gateway configured. Go to `Settings -> IPFS`, there should be an available `Set up your IPNS keys` option, which opens `brave://settings/ipfs/keys`
```

Comment:
Should this one also be `Brave local IPFS node` rather than just `Brave local node`?

Reply:
I'll change it; same answer as above.